CMakeLists.txt

```diff
@@ -163,5 +163,21 @@ ENDIF()
 #
 # And finally, print the configuration:
 #
-FILE(READ ${CMAKE_BINARY_DIR}/summary.log DEAL_II_LOG_SUMMARY)
-MESSAGE("${DEAL_II_LOG_SUMMARY}")
+FIND_PACKAGE(Perl QUIET)
+IF(PERL_FOUND AND DEAL_II_HAVE_BUNDLED_DIRECTORY)
+  EXECUTE_PROCESS(
+    COMMAND ls ${COWSAY_FOLDER}/cows
```

Comment:
Realizing now what cowsay is, how about something like "Use the bundled version of the cowsay system to produce nicer output."